### ISSUE TITLE
spyder console can not close ~

### ISSUE THREAD

**Issue by shenxiaozhong** [spyder-ide/spyder#7622]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "D:\Anaconda3\envs\sxzlm\lib\site-packages\spyder\plugins\ipythonconsole.py", line 1273, in close_client
    self.create_new_client()
  File "D:\Anaconda3\envs\sxzlm\lib\site-packages\spyder\plugins\ipythonconsole.py", line 976, in create_new_client
    interpreter=pyexec)
  File "D:\Anaconda3\envs\sxzlm\lib\site-packages\spyder\utils\programs.py", line 405, in is_module_installed
    fd, script = tempfile.mkstemp(suffix='.py', dir=TEMPDIR)
  File "D:\Anaconda3\envs\sxzlm\lib\tempfile.py", line 344, in mkstemp
    return _mkstemp_inner(dir, prefix, suffix, flags, output_type)
  File "D:\Anaconda3\envs\sxzlm\lib\tempfile.py", line 262, in _mkstemp_inner
    fd = _os.open(file, flags, 0o600)
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\fred\\AppData\\Local\\Temp\\spyder\\tmpljc8gdrd.py'

## Version and main components

* Spyder Version: 3.2.3 
* Python Version: 3.6.6
* Qt Versions: 5.9.3, PyQt5 5.9.2 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  None (NOK)
numpy >=1.7      :  1.13.1 (OK)

**Comment by ccordoba12** [comment]
I think this is fixed in our latest version (3.3.0). Please update by opening the Anaconda Prompt and running there

    conda update spyder

and try again.

**Comment by ccordoba12** [comment]
Closing because of lack of response.
